storage/remote/codec.go

```diff
@@ -540,3 +541,12 @@ func labelsToLabelsProto(labels labels.Labels, buf []prompb.Label) []prompb.Labe
 	}
 	return result
 }
+
+func metricTypeToMetricTypeProto(t textparse.MetricType) prompb.MetricMetadata_MetricType {
+	v, ok := prompb.MetricMetadata_MetricType_value[string(t)]
```

Comment:
A comment here would be useful - I'd originally assumed `MetricType` was an enum, and was surprised to see it was in fact a string.